app/containers/ReactionsList.tsx

```diff
@@ -103,35 +135,102 @@ const ReactionsTabBar = ({ tabs, activeTab, goToPage, baseUrl, getCustomEmoji, w
 
 const UsersList = ({ tabLabel }: { tabLabel: IReaction }) => {
 	const { colors } = useTheme();
-	const { emoji, usernames } = tabLabel;
+	const { emoji, usernames, names } = tabLabel;
+	const users =
+		names?.length > 0
+			? usernames.map((username, index) => ({ username, name: names[index] }))
+			: usernames.map(username => ({ username, name: '' }));
 	return (
 		<FlatList
-			data={usernames}
+			data={users}
 			ListHeaderComponent={() => (
 				<View style={styles.emojiName}>
 					<Text style={{ color: colors.auxiliaryTintColor }}>{emoji}</Text>
 				</View>
 			)}
 			renderItem={({ item }) => (
 				<View style={styles.userItemContainer}>
-					<Avatar text={item} size={36} />
-					<View style={styles.usernameContainer}>
-						<Text style={[styles.usernameText, { color: colors.titleText }]}>{item}</Text>
+					<Avatar text={item.username} size={36} />
+					<View style={styles.textContainer}>
+						{item.name ? (
+							<>
+								<Text style={[styles.name, { color: colors.titleText }]} numberOfLines={1}>
+									{item.name}
+								</Text>
+								<Text style={[styles.username, { color: colors.auxiliaryText }]} numberOfLines={1}>
+									@{item.username}
+								</Text>
```

Comment:
The idea of `useRealName` is to:
- if true, show the full name of the user. John Doe.
- If false, show their username. john.doe.

It's not supposed to show more info.

Reply:
I was planning on displaying the UserItem similar to the one used in other places in the app when `useRealName` is on:
![image](https://user-images.githubusercontent.com/77742477/184085613-69c54e7e-9513-409a-9433-f71970545e0e.png)

But okay, I got the point. I can just replace the `username` with `fname` if `useRealName` is enabled.